index.bs

```diff
@@ -3453,6 +3453,14 @@ When this operation is invoked, the [=authenticator=] MUST perform the following
             ::  return an error code equivalent to "{{NotAllowedError}}" and terminate the operation.
         </dl>
 
+        Note: The purpose of this [=authorization gesture=] is not to proceed with creating a credential,
+        but for privacy reasons to authorize disclosure of the fact that
+        <code>|descriptor|.{{PublicKeyCredentialDescriptor/id}}</code> is [=bound credential|bound=] to this [=authenticator=].
+        If the user consents, the [=client=] and [=[RP]=] can detect this and guide the user to use a different [=authenticator=].
+        If the user does not consent,
+        the [=authenticator=] does not reveal that <code>|descriptor|.{{PublicKeyCredentialDescriptor/id}}</code> is bound to it,
+        and responds as if the user simply declined consent to create a credential.
+
```

Comment:
the above looks fine, thx!  

Tho, we're saying here: "if the RP gets an `invalidStateError` back from a nav.creds.Create() call, they can do something user-helpful with that info. But this guidance for the RP is buried in "Note:"s in spec sections that we are not suggesting RP devs read.  Additionally, our "rp operations" section only says "abort the ceremony with a user-visible error" if an error is returned from Create().  Seems like we oughta do more.  Maybe we land this and submit a new issue about these other aspects of our authn-hairball-API ?

Reply:
Good comments! A new issue for that seems good to me.

Reply:
done, see issue #1331